### ISSUE TITLE
doxyrest command returns "duplicate member id group..."

### ISSUE THREAD

**Issue by mateasmario** [doxygen/doxygen#8790]
I'm generating documentation for my code using `doxygen`, `doxyrest` and `sphinx-build` commands. I'm using doxygen 1.9.2, doxyrest 2.1.2 and sphinx 3.5.4.

No warnings while using `doxygen Doxyfile`, but when I type in `doxyrest -c doxyrest-config.lua` to generate rst, based on the xml made by Doxygen, my screen gets full of errors like this one:

`warning: duplicate member id group___my_example_group_commands_1gaaa56d9b... (function: ExampleFunction)
`
I'm using \addtogroup to group more functions at a time, like this:

```
/** \addtogroup MyGroup
* @{ */

   /** \addtogroup SomeSubgroup
   * @{ */

      void someFunctionHere(int param1, int param2, ...) {
         ... // some stuff here
      }
      
      void someOtherFunction(char *param1, ...) {
         ... // some other stuff here
      }

   /** @} */

/** @} */
```
My question is: where does that `duplicate member id ...` error come from? Do you have any idea?

Thank you.

**Comment by albert-github** [comment]
This is not a site for questions about doxyrest or sphinx but as the discussion at https://stackoverflow.com/q/69264144/1657886 already revealed the problem can be seen in the doxygen part.

Example: [example.tar.gz](https://github.com/doxygen/doxygen/files/7208786/example.tar.gz)

With the example I see the problem in the combination `aa_8h.xml` and `group___some_subgroup.xml`, in the HTML and LaTeX output the problem is not present. The labels, in this case, are: group___some_subgroup_1ga2b659d1e78f341280e5247b4a3ffc26d and group___some_subgroup_1gad49b38221d81bb3f0a2703fded52091b

**Comment by katzz666** [comment]
Hey, do you have any updates on this issue?
I get the same warnings and I can't find any solution to how to fix it.
Thank you.

**Comment by albert-github** [comment]
@katzz666 
- Which version of doxygen are you using?


I just checked with the 1.9.6 version as well as the current master (1.9.7 (19420b7ae5fb764654df9d949d62429ae1432df2)) and here the problem is still visible.

**Comment by vespageorge** [comment]
I tried 1.9.6 and 1.9.5 both with same problem

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by vespageorge** [comment]
> Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

@albert-github - amazing job, thank you

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.7.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
